### PR TITLE
feat: added Typography Component

### PR DESCRIPTION
**Pull request by srv-pranjal**
# Added Typography Component
## Description
Typography

 - Headings 
 - Text Utilities
 - Colored Text Utilities
 - Aligned Text Utilities
 
Animation1

### CODE REVIEW

File: pages/typography.html [hunk 1]
```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Pranjal Srivastava" />
```
Comment (Mohasin-Haque) [inline pages/typography.html:7]: You can add meta descriptions here.

File: pages/typography.html [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+    &lt;h2 class="heading-2"&gt;Heading-2&lt;/h2&gt;
+
+    &lt;h3 class="heading-3"&gt;Heading-3&lt;/h3&gt;
+
+    &lt;h4 class="heading-4"&gt;Heading-4&lt;/h4&gt;
+
+    &lt;h5 class="heading-5"&gt;Heading-5&lt;/h5&gt;
+
+    &lt;h6 class="heading-6"&gt;Heading-6&lt;/h6&gt;
```
Comment (Mohasin-Haque) [inline pages/typography.html:140]: Did you wrote your own version of code snippet!! This looks so good.

**Review by Mohasin-Haque — COMMENTED** [review]
Your folder structure is so good and also your coding style. You can go ahead and merge it.

**Review by sakshi006 — APPROVED** [review]
Looks good to merge

File: pages/typography.html (hunk 1 above)
Comment (srv-pranjal) [reply pages/typography.html:7]: Sure, I will

File: pages/typography.html (hunk 2 above)
Comment (srv-pranjal) [reply pages/typography.html:140]: No I'm using Highlight JS

**Pull request merged**